.stylelintrc.js

```diff
@@ -18,41 +19,31 @@ module.exports = {
         'no-extra-semicolons': null, // fix GH-1832 - workaround for wikimedia/less.php parser
         'number-max-precision': 5,
         'rule-empty-line-before': null,
+        'string-quotes': 'single',
```

Comment:
Let's stick to double quotes for HTML/CSS/JSON and single quotes for JS. This will also heavily reduce the changes of this PR. Doublequotes are more prominent and readable in selectors and html attributes.

Reply:
I was testing the waters here :), I completely agree double quotes seems more natural in HTML attributes and I was supprised there is no separate config for it.

Using double quotes for LESS values is however not much natural too, so I will keep the rule turned off and commit only double quotes -> single quotes changes in LESS values manually.